src/plugins/spotifyControls/index.tsx

```diff
@@ -70,21 +70,20 @@ export default definePlugin({
                 replace: "false",
             }]
         },
-        // Discord doesn't give you the repeat kind, only a boolean
         {
             find: 'repeat:"off"!==',
-            replacement: {
-                match: /repeat:"off"!==(.{1,3}),/,
-                replace: "actual_repeat:$1,$&"
-            }
+            replacement: [
+                {
+                    // Discord doesn't give you shuffle state and the repeat kind, only a boolean
+                    match: /repeat:"off"!==(.{1,3}),/,
```

Comment:
i know this is old but always good to refactor old stuff while you're already editing it
```suggestion
                    match: /repeat:"off"!==(\i),/,
```